recipes/lua-5.1.5/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: lua
+  version: 5.1.5
+
+source:
+  fn: lua-5.1.5.tar.gz
+  url: http://www.lua.org/ftp/lua-5.1.5.tar.gz
+  md5: 2e115fe26e435e33b0d5c022e4490567
+  patches:
+    - update.patch [not win]
```

Comment:
Can you add a comment describing this patch? Also, we use the comment style for the pre-processors.

` [not win]` should be `# [not win]`